types/index.d.ts

```diff
@@ -328,10 +354,10 @@ export interface CookieOption {
 
 /** [Documentation](https://next-auth.js.org/configuration/options#cookies) */
 export interface CookiesOptions {
-  sessionToken?: CookieOption
-  callbackUrl?: CookieOption
-  csrfToken?: CookieOption
-  pkceCodeVerifier?: CookieOption
+  sessionToken: CookieOption
+  callbackUrl: CookieOption
+  csrfToken: CookieOption
+  pkceCodeVerifier: CookieOption
```

Comment:
So the `CookiesOptions` interface is marked this way now, but on line 199 you can see it is `Partial`. This is so that `InternalOptions` can still use `CookieOptions` where all properties are present (as we merge with a default object).